subprojects/gradle/ci-logger/src/main/kotlin/com/avito/utils/logging/CILoggingDestination.kt

```diff
@@ -0,0 +1,53 @@
+package com.avito.utils.logging
+
+import java.io.File
+import java.io.Serializable
+
+interface CILoggingDestination : Serializable {
+
+    fun write(message: String, throwable: Throwable?)
+    fun child(tag: String): CILoggingDestination
+}
+
+internal object StdoutDestination : CILoggingDestination {
+
+    override fun write(message: String, throwable: Throwable?) {
+        println(message)
+        println(throwable)
+    }
+
+    override fun child(tag: String): CILoggingDestination = this
+}
+
+internal object OnlyMessageStdoutDestination : CILoggingDestination {
```

Comment:
This is one of key changes of that PR. Before we can't print exceptions explicitly for critical and warn levels